src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -272,11 +276,12 @@ public Response getAssigned(@Context final HttpServletRequest request,
                 // In order to get all the information about gameboard items, we need to use the method which augments
                 // gameboards with user attempt information. But we don't _want_ this information for real, so we won't
                 // do the costly loading of the real attempt information from the database:
+                // FIXME: would GameManager::getGameboards with only one argument work correctly / be better?
```

Comment:
Minor, but I think this has potential to become an immortal 'FIXME' - is it worth following up before we merge?

Reply:
Okay, I followed it up and the answer is "no". So I have updated the comment to better explain _why_ and removed the FIXME :P